src/main/java/github/CollaboratorServiceEx.java

```diff
@@ -0,0 +1,47 @@
+package github;
+
+import org.apache.logging.log4j.Logger;
+import org.eclipse.egit.github.core.IRepositoryIdProvider;
+import org.eclipse.egit.github.core.User;
+import org.eclipse.egit.github.core.client.GitHubClient;
+import org.eclipse.egit.github.core.service.CollaboratorService;
+import org.eclipse.egit.github.core.service.UserService;
+import util.HTLog;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CollaboratorServiceEx extends CollaboratorService {
```

Comment:
I think so. All the services that we extend from existing services in `egit.github` have a `Ex` suffix.